CMakeLists.txt

```diff
@@ -234,6 +242,13 @@ if (NOT roscpp_FOUND)
   list(APPEND plugins gazebo_geotagged_images_plugin)
 endif()
 
+if (BUILD_ROS_INTERFACE)
+  add_library(rotors_gazebo_motor_fail_plugin SHARED src/gazebo_motor_failure_plugin.cpp)
```

Comment:
@siddhu45 now that I see, that's not the name of the plugin. It should be `gazebo_motor_fail_plugin` and not `rotors_gazebo_motor_fail_plugin` (that's related to `rotors_gazebo` package).

Reply:
oh, yes!
I will amend it.